tests/define_group_test_project.py

```diff
@@ -0,0 +1,87 @@
+import flow
+from flow import FlowProject
```

Comment:
I think it would make more sense to just extend the original test project with a few grouped operations instead of creating a completely separate one. That would also be a very good test whether we are introducing backwards incompatible changes.